jsonschema-validation.xml

```diff
@@ -539,27 +557,141 @@
+                        Note in particular that the requirement is limited to syntactic checking; it is
+                        not to be expected that an implementation would send an email, attempt to connect
+                        to a URL, or otherwise check the existence of an entity identified by a format
+                        instance.
+                        <cref>
+                            The expectation is that for simple formats such as date-time, syntactic
+                            validation will be thorough.  For a complex format such as email addresses,
+                            which are the amalgamation of various standards and numerous adjustments
+                            over time, with obscure and/or obsolete rules that may or may not be
+                            restricted by other applications making use of the value, a minimal validation
+                            is sufficient.  For example, an instance string that does not contain
+                            an "@" is clearly not a valid email address, and an "email" or "hostname"
+                            containing characters outside of 7-bit ASCII is likewise clearly invalid.
+                        </cref>
+                    </t>
+                    <t>
+                        It is RECOMMENDED that implementations use a common parsing library for each format,
+                        or a well-known regular expression.  Implementations SHOULD clearly document
+                        how and to what degree each format attribute is validated.
+                    </t>
+                    <t>
+                        The <xref target="meta-schema">standard core and validation meta-schema</xref>
+                        includes this vocabulary in its "$vocabulary" keyword with a value of false,
+                        since by default implementations are not required to support this keyword
+                        as an assertion.  Supporting the format vocabulary with a value of true is
+                        understood to greatly increase code size and in some cases execution time,
+                        and will not be appropriate for all implementations.
+                    </t>
+                </section>
+                <section title="Custom format attributes">
+                    <t>
+                        Implementations MAY support custom format attributes. Save for agreement between
+                        parties, schema authors SHALL NOT expect a peer implementation to support such
+                        custom format attributes.  An implementation MUST NOT fail
+                        validation or cease processing due to an unknown format attribute.
+                        When treating "format" as an annotation, implementations SHOULD collect both
+                        known and unknown format attribute values.
```

Comment:
"We" should build some tests around collecting format values as annotations.

Reply:
Yeah.  And really around annotation collection in general.

Reply:
Do we have an issue to track this?

Reply:
@philsturgeon no, but feel free to file one 😄